docs/nodes/qgb-deploy.md

```diff
@@ -0,0 +1,82 @@
+# Deploy the QGB contract
+
+<!-- markdownlint-disable MD013 -->
+
+The `deploy` is a helper command that allows deploying the QGB smart contract to a new EVM chain:
+
+```ssh
+qgb deploy --help
+
+Deploys the QGB contract and initializes it using the provided Celestia chain
+
+Usage:
+  qgb deploy <flags> [flags]
+  qgb deploy [command]
+
+Available Commands:
+  keys        QGB keys manager
+
+Flags:
+  -c, --celes-grpc string       Specify the grpc address (default "localhost:9090")
+  -d, --evm-address string      Specify the EVM account address to use for signing (Note: the private key should be in the keystore)
+  -z, --evm-chain-id uint       Specify the evm chain id (default 5)
+  -l, --evm-gas-limit uint      Specify the evm gas limit (default 25000000)
+      --evm-passphrase string   the evm account passphrase (if not specified as a flag, it will be asked interactively)
+  -e, --evm-rpc string          Specify the ethereum rpc address (default "http://localhost:8545")
+  -h, --help                    help for deploy
+      --home string             The qgb deployer home directory
+  -n, --starting-nonce string   Specify the nonce to start the QGB contract from. "earliest": for genesis, "latest": for latest valset nonce, "nonce": for the latest valset before the provided nonce, provided nonce included. (default "latest")
+
+Use "qgb deploy [command] --help" for more information about a command.
+```
+
+## How to run
+
+### Install the QGB binary
+
+Make sure to have the QGB binary installed. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/README.md) for more details.
```

Comment:
let's make a task for a future PR to show how to install the binary inside the docs so folks don't leave the docs. not urgent for now.